src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/ArrayTrailingCommaCheckTest.java

```diff
@@ -45,7 +45,6 @@ public void testDefault()
             createCheckConfig(ArrayTrailingCommaCheck.class);
         final String[] expected = {
             "17: " + getCheckMessage(MSG_KEY),
-            "34: " + getCheckMessage(MSG_KEY),
             "37: " + getCheckMessage(MSG_KEY),
```

Comment:
Violation is reported for line which contains right curly of the inspected array, therefore an exception is expected on line 37.